Licensing_Plugin_Admin.php

```diff
@@ -354,14 +354,16 @@ private function get_cdn_notice( $status, $license_key ) {
 				$billing_url = $this->get_billing_url( $license_key );
 				if ( $billing_url ) {
 					return sprintf(
-						// Translators: 1 Total CDN tradmark, 2 opening HTML a tag to billing portal, 3 closing HTML a tag.
+						// Translators: 1 Total CDN tradmark, 2 opening HTML a tag to billing portal
+						// Translators: 3 closing HTML a tag, 4 button to refresh license status.
 						__(
-							'Your %1$s subscription payment is past due. Please update your %2$sBilling Information%3$s to prevent service interruption',
+							'Your %1$s subscription payment is past due. Please update your %2$sBilling Information%3$s to prevent service interruption. Once your billing information has been updated and payment is successful please manually %4$s otherwise it may take up to 1 hour to refresh on it\'s own.',
```

Comment:
Grammer: This should be its own not it's own

Reply:
Updated